pkg/kubelet/remote/remote_runtime.go

```diff
@@ -82,7 +85,16 @@ func (r *RemoteRuntimeService) Version(apiVersion string) (*runtimeapi.VersionRe
 func (r *RemoteRuntimeService) RunPodSandbox(config *runtimeapi.PodSandboxConfig) (string, error) {
 	// Use 2 times longer timeout for sandbox operation (4 mins by default)
 	// TODO: Make the pod sandbox timeout configurable.
-	ctx, cancel := getContextWithTimeout(r.timeout * 2)
+	timeout := r.timeout
```

Comment:
I am wondering do we really need to make this configurable?

Sandbox is indispensable for pods. Sandbox is a small image that is expected to be running well. I think a default value is enough.

/cc @kubernetes/sig-node-pr-reviews 
/cc @mtaufen PTAL.